src/applications/mhv-landing-page/tests/e2e/cerner-facility-info-alert/cerner-facility-info-alert.cypress.spec.js

```diff
@@ -0,0 +1,64 @@
+import manifest from '../../../manifest.json';
+import { generateUser } from '../../../mocks/api/user';
+import ApiInitializer from '../utilities/ApiInitializer';
+import LandingPage from '../pages/LandingPage';
+
+describe(manifest.appName, () => {
+  describe('Cerner facility info alert', () => {
+    beforeEach(() => {
+      ApiInitializer.initializeFeatureToggle.withCurrentFeatures();
+    });
+
+    it('renders info alert and learn more link when userFacilityReadyForInfoAlert is true', () => {
+      const oracleHealthUser = generateUser({ oracleHealth: true });
+      LandingPage.visit({ user: oracleHealthUser });
+
+      // Info alert should be visible
+      cy.findByTestId('cerner-facilities-info-alert').should('exist');
+
+      // Expand the alert by clicking on it
+      cy.findByTestId('cerner-facilities-info-alert').click();
+
+      // Verify expanded content is visible
+      cy.findByTestId('cerner-facility-info-text').should('be.visible');
+      cy.contains('brought all your VA health care data together').should(
+        'be.visible',
+      );
+      cy.contains('Still want to use My VA Health for now?').should(
```

Comment:
use `findByText` instead. Always try to follow the preferences from the developer docs with any `find` method as preferred. If this is a shadow DOM then it may not apply.